inc/class-cwpbp.php

```diff
@@ -92,10 +92,10 @@ function bpSignUpMeta( $usermeta ) {
 	 */
 	function bpActivated( $user_id, $key, $user ) {
 
-		$tag = 'notify';
+		$tag = 'cwp_bp_notify';
```

Comment:
sign up form value was not being saved due to targeting non-existent `$user['meta']` value: 

![screen shot 2018-04-18 at 3 51 05 pm](https://user-images.githubusercontent.com/2048170/38962384-e26aae24-4321-11e8-89e2-dd0a14563bb4.png)

Reply:
nice catch